gatsby/content/docs/moderation-guide.mdx

```diff
@@ -190,12 +190,12 @@ If you're using Element, when you go to view more information about a particular
 ![Element-deactivate-btn](/docs/projects/images/moderation8.png)
 
 
-Removing abusive rooms is done by calling the /shutdown_room API, e.g:
+Removing abusive rooms is done by calling the /DELETE API, e.g:
```

Comment:
Thanks for taking care of this! I would tend to avoid using `/DELETE` as it makes it look like an actual endpoint. I believe you have https://matrix-org.github.io/synapse/latest/admin_api/rooms.html#delete-room-api in mind?

What about the following?
```suggestion
Removing abusive rooms is done by calling the delete room API, e.g:
```